dockerfiles/init/modules/keycloak/files/che/login/messages/messages_en.properties

```diff
@@ -0,0 +1,240 @@
+product=Eclipse Che
```

Comment:
Do we need to replace our current messages with messages from new upstream releases?
I think we have our own Che theme and we shouldn't depend on changes in upstream themes.